packages/libs/react-ui/src/components/Typography/Heading/Heading.css.ts

```diff
@@ -13,68 +13,68 @@ export const elementVariants = {
   h1: [
     sprinkles({
       fontSize: {
-        xs: '5xl',
-        md: '7xl',
-        lg: '9xl',
-        xl: '10xl',
-        '2xl': '12xl',
+        xs: '$5xl',
+        md: '$7xl',
+        lg: '$9xl',
+        xl: '$10xl',
+        xxl: '$12xl',
       },
     }),
   ],
   h2: [
     sprinkles({
-      fontWeight: 'bold',
+      fontWeight: '$bold',
       fontSize: {
-        xs: '4xl',
-        lg: '6xl',
-        xl: '8xl',
-        '2xl': '9xl',
+        xs: '$4xl',
```

Comment:
why do we have `$4` as well as `$4xl`?

Reply:
when to use `$4` vs `$md`?

Reply:
So spacing tokens have the following naming conventions:
- Numeric - the number represents the number * 4 (our design system is built on units of 4). This size is fixed 
- xs, sm, md, etc.. - which are spacing variables that change depending on the breakpoints. These are the values that are used in layout components, but can be used anywhere

Fontsizes have a relative size based naming convention so they would have the following result assuming that the default browser fontsize (16px) is not changed:
$sx = 12px
$sm = 14px
$base = 16px
$md = 18px
$lg  = 20px
...etc 
$12xl = 84px

I can see how this is a bit confusing; this is just the naming convention that I felt worked the best. Does this make sense or do you have any other suggestions?

Reply:
I understand the numeric and md/sm etc, thanks. I still wonder what `$4xl` falls into though.

Reply:
`$4xl` would fall into the fontSizes category since they are relative to the base. The options would be 
```
  fontSizes: {
    xs: '0.75rem', // 12px
    sm: '0.875rem', // 14px
    base: '1rem', // 16px
    md: '1.123rem', // 18px
    lg: '1.25rem', // 20px
    xl: '1.5rem', // 24px
    '2xl': '1.75rem', // 28px
    '3xl': '2rem', // 32px
    '4xl': '2.25rem', // 36px
    '5xl': '2.5rem', // 40px
    '6xl': '2.75rem', // 44px
    '7xl': '3rem', // 48px
    '8xl': '3.25rem', // 52px
    '9xl': '3.75rem', // 60px
    '10xl': '4.5rem', // 72px
    '11xl': '5rem', // 80px
    '12xl': '5.25rem', // 84px
  },
```

Maybe in the future if we notice we are only using certain ones, we can decrease the options